bin/restart

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH.unshift File.expand_path('../../', __FILE__)
+
+require 'config/initializers/figaro'
+require 'winrm'
+
+options = {
+  endpoint: format(
+    'http://%s:%s/wsman',
```

Comment:
I tried using SSL and received the following error:

```
/Users/josephbridgwaterrowe/.rvm/gems/ruby-2.2.2/gems/httpclient-2.8.3/lib/httpclient/ssl_socket.rb:103:in `connect': SSL_connect returned=1 errno=0 state=SSLv2/v3 read server hello A: unknown protocol (OpenSSL::SSL::SSLError)
```

I'm going to see if we can get SSL enabled on it... to support this in future I will add a setting for protocol.